### PR TITLE
feat(widget): add awsDataTransferByRegion widget

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
- update map-chart-helper
- add AWSDataTransferByRegion widget

스크린샷 2022-12-02 오후 12 34 29

### Things to Talk About
There's something trouble applying color. 🥲

### CODE REVIEW

File: src/services/dashboards/widgets/aws-data-transfer-by-region/AWSDataTransferByRegion.vue [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    const chart = createMapChart();
+    const polygonSeries = createMapPolygonSeries();
+    chart.series.push(polygonSeries);
+    const pointSeries = createMapPointSeries({
+        calculateAggregates: true,
+        valueField: 'value',
+        // polygonIdField: 'usage_quantity',
+    });
+    chart.series.push(pointSeries);
+    setChartColors(chart, state.colorSet); // TODO: not working...
```
Comment (yuda110) [inline src/services/dashboards/widgets/aws-data-transfer-by-region/AWSDataTransferByRegion.vue:121]: this is not working yet

**Review by song2kim — APPROVED** [review]
LGTM!

**Review by minseolee — APPROVED** [review]
LGTM ! Nice Types !!

File: src/common/composables/amcharts5/map-chart-helper.ts [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
-    return chart;
-};
+export const createMapChart = (root: Root, settings?: IMapChartSettings): am5map.MapChart => root.container.children.push(am5map.MapChart.new(root, {
+    projection: am5map.geoMercator(),
+    ...settings,
+}));
+
+export const createMapPolygonSeries = (root: Root, settings?: IMapPolygonSeriesSettings): am5map.MapPolygonSeries => am5map.MapPolygonSeries.new(root, {
+    geoJSON: am5geodata_worldLow,
+    exclude: ['AQ'],
```
Comment (WANZARGEN) [inline src/common/composables/amcharts5/map-chart-helper.ts:16]: question: what is AQ?

**Review by WANZARGEN — COMMENTED** [review]
LGTM!

File: src/common/composables/amcharts5/map-chart-helper.ts (hunk 2 above)
Comment (yuda110) [reply src/common/composables/amcharts5/map-chart-helper.ts:16]: It's Antarctica :)

**Pull request merged**